libs/platform/core/src/services/page-head/page-head.service.ts

```diff
@@ -0,0 +1,16 @@
+import { ElementAttributes, ElementDefinition } from './page-head.model';
+
+export const PageHeadService = 'oryx.PageHeadService';
+
+export interface PageHeadService {
+  addElements(definitions: ElementDefinition | ElementDefinition[]): void;
```

Comment:
addTag still works better for me here (for link, script, title or meta, all of those are tags) than 'element'

Reply:
or maybe just `add` and that's it?

Reply:
looks like title, link, etc. are also considered meta-related elements (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta), then 
```
export interface PageMetaService {
  add(definition: MetaElement | MetaElement[])
  // ...
```
looks quite concise?

Reply:
But we decided them PageMeta incorrect name. Cause we can add also title\script which is not meta things

Reply:
ok, I'll fix it